packages/editor/src/lib/components/Shape.tsx

```diff
@@ -156,7 +158,13 @@ export const Shape = memo(function Shape({
 					</OptionalErrorBoundary>
 				</div>
 			)}
-			<div ref={containerRef} className="tl-shape" data-shape-type={shape.type} draggable={false}>
+			<div
+				ref={containerRef}
+				className="tl-shape"
+				data-shape-type={shape.type}
+				data-shape-is-filled={isFilledShape}
+				draggable={false}
+			>
```

Comment:
Do we need to set pointer-events: all at the shape level? I'd prefer not to do this and have the contents of the shape determine whether they have pointer events. Sorry if turning on pointer events on a shape level was introduced earlier, I must have missed it. Wouldn't we turning on pointer events for the text area be sufficient?

Reply:
Yeah, hah, that's actually the _other_ PR that was stacked on top of this one: https://github.com/tldraw/tldraw/pull/3578

I think maybe you were commenting about the changes made in that other PR?

(I was also confused for a sec, because we were looking on staging, but this PR hasn't been pushed to staging yet)